contract_price_revision/tests/test_contract_price_revision.py

```diff
@@ -82,3 +82,28 @@ def test_contract_price_revision_invoicing(self):
         self.assertEqual(len(lines), 2)
         lines = lines.filtered(lambda x: x.price_unit == 200.0)
         self.assertEqual(len(lines), 1)
+
+    def test_compute_variation_percent(self):
+        self.acct_line.price_unit = 200.0
+        self.acct_line.previous_price = 100.0
+        self.acct_line._compute_variation_percent()
+        self.assertEqual(self.acct_line.variation_percent, 100.0)
+        self.acct_line.price_unit = 200.0
+        self.acct_line.previous_price = 0.0
+        self.acct_line._compute_variation_percent()
+        self.assertEqual(self.acct_line.variation_percent, 0.0)
+
+    def test_get_default_variation_type(self):
+        variation_type = self.env[
+            "contract.price.revision.wizard"
+        ]._get_default_variation_type()
+        self.assertEqual(variation_type, "percentage")
```

Comment:
This shouldn't be tested this way, but on a real record creation. You can do the assert after this line: https://github.com/OCA/contract/pull/1105/commits/5c1275660b691830945fa16a8fa8fa7e573815ab#diff-9ac09ee293d9ab722c74e65e44634967a47cfb37f8ec845d6f0113c3d9e1961bR63 (and no need for an extra test).

Reply:
Done!